libs/net/kafka/KafkaListener`.cs

```diff
@@ -200,6 +205,7 @@ public void Stop()
             this.IsConsuming = false;
             this.IsReady = false;
             this.Consumer.Close();
+            OnStop?.Invoke(this, new EventArgs());
```

Comment:
Inform the owner of this object it has stopped.  This way the owner can cancel the thread.